--- a/runtime/Cpp/CMakeLists.txt
+++ b/runtime/Cpp/CMakeLists.txt
@@ -1,5 +1,5 @@
 # -*- mode:cmake -*-
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 3.0)
 # 2.8 needed because of ExternalProject

 # Detect build type, fallback to release and throw a warning if use didn't specify any
@@ -37,10 +37,10 @@ if(CMAKE_VERSION VERSION_EQUAL "3.3.0" OR
   CMAKE_POLICY(SET CMP0054 OLD)
 endif()

-if(CMAKE_SYSTEM_NAME MATCHES "Linux")
-  find_package(PkgConfig REQUIRED)
-  pkg_check_modules(UUID REQUIRED uuid)
-endif()
+#if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+#  find_package(PkgConfig REQUIRED)
+#  pkg_check_modules(UUID REQUIRED uuid)
+#endif()
 if(APPLE)
   find_library(COREFOUNDATION_LIBRARY CoreFoundation)
 endif()
 